3rdparty/python/requirements.txt

```diff
@@ -38,13 +38,6 @@ node-semver==0.9.0
 # This dependency is only for debugging Pants itself, and should never be imported
 debugpy==1.6.0
 
-# These dependencies must only be used from the explorer backend, and no code outside that backend
-# may import anything from it, so these libraries are not ending up as requirements of Pants itself.
-fastapi==0.78.0
-starlette==0.19.1
-strawberry-graphql[fastapi]==0.114.0
```

Comment:
These moved to `pants-plugins/pants_explorer/server/requirements.txt` which is supported by https://github.com/pantsbuild/pants/pull/19406